### PR TITLE
Adds getMinimumBalanceForRentExemption , getEpochInfo, getTransactionCount, getStakeActivation, getInflationGovernor methods

### PR DESCRIPTION
**Pull request by OmkarAcharekar**
The pr adds 5 requests of  issue #39 

* [getMinimumBalanceForRentExemption](https://solana-labs.github.io/solana-web3.js/classes/Connection.html#getMinimumBalanceForRentExemption)
* [getEpochInfo ](https://solana-labs.github.io/solana-web3.js/classes/Connection.html#getEpochInfo)
* [getTransactionCount](https://solana-labs.github.io/solana-web3.js/classes/Connection.html#getTransactionCount)
* [getStakeActivation](https://solana-labs.github.io/solana-web3.js/classes/Connection.html#getStakeActivation)
* [getInflationGovernor](https://solana-labs.github.io/solana-web3.js/classes/Connection.html#getInflationGovernor)

### CODE REVIEW

File: tests/main.cpp [hunk 1]
```diff
@@ -788,3 +788,56 @@ TEST_CASE("getEpochSchedule") {
...
+  CHECK_GE(inflationgovernor.taper, 0);
+  CHECK_GE(inflationgovernor.terminal, 0);
+}
+
+TEST_CASE("getTransactionCount") {
+  const auto connection = solana::rpc::Connection(solana::DEVNET);
+  const auto TransactionCount = connection.getTransactionCount(
+      solana::GetSlotConfig{solana::Commitment::FINALIZED});
+  CHECK_GT(TransactionCount, 0);
+  sleep(2);
```
Comment (godmodegalactus) [inline tests/main.cpp:821]: May be create some transactions confirm them and test again instead of waiting for 2 s.

File: tests/main.cpp [hunk 2]
```diff
@@ -788,3 +788,56 @@ TEST_CASE("getEpochSchedule") {
...
+  const auto TransactionCount2 = connection.getTransactionCount(
+      solana::GetSlotConfig{solana::Commitment::FINALIZED});
+  CHECK_GE(TransactionCount2, TransactionCount);
```
Comment (godmodegalactus) [inline tests/main.cpp:824]: Then you can use CHECK_GT

File: tests/main.cpp [hunk 3]
```diff
@@ -788,3 +788,56 @@ TEST_CASE("getEpochSchedule") {
...
+}
+
+TEST_CASE("getEpochInfo") {
+  const auto connection = solana::rpc::Connection(solana::DEVNET);
+  const auto epochInfo = connection.getEpochInfo();
+  CHECK_GE(epochInfo.absoluteSlot, 0);
```
Comment (godmodegalactus) [inline tests/main.cpp:830]: Always use GT please. 
`shotsInEpoch` is constant you can calculate us CHECK_EQ

File: tests/main.cpp [hunk 4]
```diff
@@ -788,3 +788,56 @@ TEST_CASE("getEpochSchedule") {
...
+         stakeactivation.state == "inactive" ||
+         stakeactivation.state == "activating" ||
+         stakeactivation.state == "deactivating"));
+}
+
+TEST_CASE("getInflationGovernor") {
+  const auto connection = solana::rpc::Connection(solana::DEVNET);
+  const auto inflationgovernor = connection.getInflationGovernor(
+      solana::commitmentconfig{solana::Commitment::FINALIZED});
+  CHECK_GE(inflationgovernor.foundation, 0);
```
Comment (godmodegalactus) [inline tests/main.cpp:809]: Use GT, some of these values should be constant, check these values const values with EQ

**Review by godmodegalactus — CHANGES_REQUESTED** [review]

File: tests/main.cpp [hunk 5]
```diff
@@ -788,3 +788,56 @@ TEST_CASE("getEpochSchedule") {
       epochschedule2.getLastSlotInEpoch(16),
       epochschedule2.firstNormalSlot + 3 * epochschedule2.slotsPerEpoch - 1);
 }
+
+TEST_CASE("stakeactivation") {
+  const auto connection = solana::rpc::Connection(solana::DEVNET);
+  const auto stakeactivation =
+      connection.getStakeActivation(solana::PublicKey::fromBase58(
```
[the remark this answers is not in the document]
Comment (godmodegalactus) [reply tests/main.cpp:795]: @mschneider what do you think about this?

File: tests/main.cpp (hunk 1 above)
Comment (OmkarAcharekar) [reply tests/main.cpp]: Done

File: tests/main.cpp (hunk 2 above)
Comment (OmkarAcharekar) [reply tests/main.cpp]: Done

File: tests/main.cpp (hunk 3 above)
Comment (OmkarAcharekar) [reply tests/main.cpp]: Done the changes

File: tests/main.cpp (hunk 4 above)
Comment (OmkarAcharekar) [reply tests/main.cpp:809]: Done ! `inflationgovernor.taper` & `inflationgovernor.terminal` seems to be constant

**Review by godmodegalactus — APPROVED** [review]

**Pull request merged**